apps/dolly-frontend/src/main/js/src/components/bestillingsveileder/stegVelger/steg/steg3/Steg3.js

```diff
@@ -54,7 +54,7 @@ export const Steg3 = ({ formikBag, brukertype, brukerId }) => {
 					alleredeValgtMiljoe={alleredeValgtMiljoe()}
 				/>
 			)}
-			{importTestnorge && <VelgGruppe formikBag={formikBag} />}
+			{visVelgGruppe && <VelgGruppe formikBag={formikBag} />}
```

Comment:
Burde nok fremdeles vise hvilket gruppe som er valgt, eller sette innsendt gruppeid som allerede valgt i VelgGruppe

Reply:
For hvis du ikke velger noen egenskaper så blir det bare vist en tom side i siste steg, som kan bli litt forvirrende.